src/ocbs/serve.py

```
@@ -0,0 +1,919 @@
+                    # Find absolute paths for commands
+                    systemctl_path = shutil.which('systemctl')
+                    openclaw_path = shutil.which('openclaw')
+                    
+                    if not systemctl_path or not openclaw_path:
+                        server_instance.logger.error("Could not find systemctl or openclaw")
+                        self.send_error(500, "Restart not available")
+                        return
+                    
+                    try:
+                        # Restart the gateway
+                        result = subprocess.run(
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)